articles/adding-dark-theme-to-your-site/index.md

```diff
@@ -0,0 +1,178 @@
+The dark theme has gained prevalence in screens today. With this trend in iOS, macOS, windows, and google, most systems adopt dark themes. Application of dark theme in website context make your website more exciting and attractive to users who love dark them. The experience is even better if you include functionality to switch between light and dark themes. This article will help you learn how to implement switching between light and dark themes using [CSS Varibles](https://www.w3schools.com/css/css3_variables.asp). Click [this](https://replit.com/@PhinaKersly/dark-theme#index.html) link to find the source code and a runnable program for the implememntation.
+
+Before we begin, we will need the following:
+### Prerequisites
+1. Your favorite text editor. I Prefer vscode
+2. Basic Knowledge of HTML, SCSS, and Javascript
+3. Understand CSS variables
+
```

Comment:
Can you add a `Table of contents` section? Something like [this](https://www.section.io/engineering-education/introduction-to-adafruit-io/#table-of-contents)